components/features/Signup/AssesseeSignup/AssesseePassword/AssesseePassword.tsx

```diff
@@ -0,0 +1,81 @@
+import { Button } from '@components/shared/elements/Button';
+import { InputField } from '@components/shared/forms/InputField';
+import { PasswordField } from '@components/shared/forms/PasswordField';
+import { SigninNotice } from '@components/shared/forms/SigninNotice';
+import { OdiLogo } from '@components/shared/svg/OdiLogo';
+import { useAssesseeSignupStoreContext } from '@context/Signup/AssesseeSignupStoreContext';
+import { confirmPasswordValidator } from '@utils/validators/confirmPasswordValidator';
+import { emailValidator } from '@utils/validators/emailValidator';
+import { passwordValidator } from '@utils/validators/passwordValidator';
+import React, { useEffect, useMemo, useRef } from 'react';
+
+import styles from "./AssesseePassword.module.css";
+
+const AssesseePassword = () => {
+    const { data, errors, setValue, setError, postResult } = useAssesseeSignupStoreContext();
+    const { password, confirmed_password: confirmedPassword, email } = data;
+
+    const emailRef = useRef<HTMLInputElement>(null);
+    const passwordRef = useRef<HTMLInputElement>(null);
+    const confirmedPasswordRef = useRef<HTMLInputElement>(null);
+  
+    const inputRefs = useMemo(() => ({
+      email: emailRef,
+      password: passwordRef,
+      confirmed_password: confirmedPasswordRef
+    }), [])
+  
+    useEffect(() => {
```

Comment:
I see this effect on the AssesseeDetails component as well, are you going to consider extracting this to a reusable hook? Maybe it could be for future refactoring tasks. Let me know :D

Reply:
Hello rashad,
I'm planning to do just exactly that. But I guess, for now, useAutoFocus can wait, we must ensure the main functionalities of the app is completed first. 😁